src/mixpanelMiddleware.js

```diff
@@ -46,6 +47,10 @@ export default function mixpanelMiddleware(token, options = {}) {
             });
         }
 
+        if (timeEvent && timeEvent.length > 0) {
```

Comment:
It looks like `timeEvent` just takes a string so you can drop the length check as an empty string will evaluate to false anywho. 

_looks like increment below is incorrect as well, looking at the documentation it can also accept an object. I'll create a separate issue for this_